index.rb

```diff
@@ -0,0 +1,43 @@
+@morse_code = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}
+
+def decode_morse(code)
+  @words = code.split('   ')
+  @words.each do |word|
+    @chars = word.split
+    @letters = ' '
+    @chars.each do |c|
+      @decode = @morse_code[c]
+      @letters += @decode
+    end
+    print @letters.to_s
+  end
+end

```

Comment:
- [ ] From the requirements you should have three methods. Kindly split this method into three based on the following requirements:

  - The first one should decode ONE morse character.
  - The second one should decode A WORD using the first method.
  - The last method should decode A SENTENCE using the second method

- [ ] Note that each method should `return` its decoded message, not `print` it (as you have done here) -  so the method can be reused